node-backend/app/controllers/usersController.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../models/sequelize");
+const jwtToken = require("../jwtTokenValidation/jwtValidation");
+module.exports.findUsersByOrg = (req, res) => {
+  let token = jwtToken.getData(req, res);
```

Comment:
use callback since it is not guaranteed that token.getData will execute immediately